src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1002.UrisMustBeValid_Invalid.sarif

```diff
@@ -110,31 +110,6 @@
             }
           ]
         },
-        {
-          "ruleId": "SARIF1002",
```

Comment:
>"ruleId": "SARIF1002", [](start = 10, length = 22)

due to skimmer fix, this node simply moved to the bottom, no change in functionality.